molgenis-data/src/main/java/org/molgenis/data/support/DefaultEntityMetaData.java

```diff
@@ -672,9 +672,14 @@ public void onChange(Package package_)
 	@Override
 	public AttributeMetaData getLabelAttribute(String languageCode)
 	{
-		String i18nLabelAttrName = getCachedAllAttrs().keySet().stream().filter(
-				attributeName -> attributeName.endsWith('-' + languageCode)).findFirst().orElse(null);
-		return i18nLabelAttrName != null ? getCachedAllAttrs().get(i18nLabelAttrName) : getLabelAttribute();
+		AttributeMetaData labelAttr = getLabelAttribute();
+		String labelAttributeName = labelAttr.getName();
+		if(labelAttributeName.endsWith("-en"))
```

Comment:
use "-"+LanguageService.FALLBACK_LANGUAGE to prevent the functionality from breaking when we change the default language